package.json

```diff
@@ -25,24 +25,24 @@
     "url": "https://github.com/uxsolutions/bootstrap-datepicker.git"
   },
   "dependencies": {
-    "jquery" : ">=1.7.1 <4.0.0"
+    "jquery": ">=1.7.1 <4.0.0"
   },
   "devDependencies": {
-    "grunt": "~0.4.5",
+    "grunt": "^1.0.1",
     "grunt-banner": "~0.6.0",
-    "load-grunt-tasks": "~1.0.0",
-    "grunt-contrib-jshint": "~0.12.0",
-    "grunt-contrib-qunit": "~0.5.2",
-    "grunt-contrib-less": "~1.3.0",
-    "grunt-contrib-compress": "~0.13.0",
-    "grunt-contrib-uglify": "~0.6.0",
-    "grunt-contrib-csslint": "~0.3.1",
-    "grunt-contrib-cssmin": "~0.14.0",
-    "grunt-contrib-clean": "~0.6.0",
-    "grunt-contrib-concat": "~0.5.0",
-    "grunt-jscs": ">=1.1.0, <2.0",
-    "grunt-string-replace": "~1.0.0",
-    "time-grunt": "~1.0.0",
-    "phantomjs-prebuilt": "~2.1.7"
+    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-compress": "^1.4.1",
+    "grunt-contrib-concat": "^1.0.1",
+    "grunt-contrib-csslint": "^2.0.0",
+    "grunt-contrib-cssmin": "^1.0.2",
+    "grunt-contrib-jshint": "^1.1.0",
+    "grunt-contrib-less": "^1.4.0",
+    "grunt-contrib-qunit": "^1.2.0",
+    "grunt-contrib-uglify": "^2.0.0",
+    "grunt-jscs": "^3.0.1",
+    "grunt-string-replace": "^1.3.1",
+    "load-grunt-tasks": "^3.5.2",
+    "phantomjs-prebuilt": "~2.1.7",
```

Comment:
[Tilde range](https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1) matches all higher patch version as well. `~2.1.7` matches `2.1.x` if x is greater or equal 7.